### ISSUE TITLE
Missing sample json config files break colab demo

### ISSUE THREAD

**Issue by JacobARose** [mle-infrastructure/mle-logging#2]
Hello!

Just read your blogpost and ~50% of the way through the colab demo, and I have to say that so far it looks like this project has the potential to be profoundly clarifying in how it simplifies & abstracts various pieces of key experiment logic that otherwise suffers from unnecessary complexity. As a PhD student who has had to refactor my whole experimental configuration workflow more times than I would like to admit to even myself, I'm super excited to try out your logger!

I'd also like to commend you for how to-the-point your choice of explanatory examples were for the blogpost. Too many frameworks fill their docs with a bunch of overly-simplistic toy problems and fail to bridge the gap between these and a real experimental situation (e.g. the elegant layout of your multi-seed, multi-config experiment

That said, my experience working through your demo was interrupted once I reached the section "Log Different Random Seeds for Same Configuration". It seems this code cell references a file called "config_1.json", which doesnt exist. While I'm sure I could figure out a simple json file with 1-2 example items, this kind of guesswork distracts immensely from the otherwise very elegant flow from simple to complex that you've set up. I also assume your target audience stretches further than experienced coders, so providing a simple demo config file to reduce the time from reading->coding seems worthwhile.

tldr; the colab needs 1-2 demo config json files

**Comment by RobertTLange** [comment]
Dear Jacob,
Thank you very much for the kind words and bringing this up. I have just fixed the issue in the notebook. I forgot that you need to download the configuration files if the notebook is executed from a Colab VM. Sorry for the inconvenience. Please let me know if you have any other recommendations/feature requests that could make your life easier.

@ configuration-file-layout: As long as your configuration file is a `.json` or `.yaml` the logger will load, copy it to the experiment log directory and internally store the resulting dictionary as a string. Alternatively, I added a feature so that you can also explicitly provide a dictionary (with arbitrary nesting) as an input to the logger: 

```python
log = MLELogger(time_to_track=['num_updates', 'num_epochs'],
                what_to_track=['train_loss', 'test_loss'],
                experiment_dir="experiment_dir/",
                model_type='torch',
                config_dict={"lrate": 0.01,
                             "momentum": 0.9})
```

The logger will then instead store the dictionary. But you are right, I will add a note about this somewhere.
Have a great Friday and again thank you,
Rob
